src/styles/block/watch.scss

```diff
@@ -0,0 +1,46 @@
+.watch {
+  width: $watchSize;
+  height: $watchSize;
+  border-radius: 50%;
+  background-color: rgb(255, 233, 92);
+  border: 4px solid rgb(0, 0, 0);
+  cursor: pointer;
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+    transform: translate(-50%, -50%);
+    background-color: black;
+    z-index: 4;
+  }
+
+  &__hand-min {
+    position: absolute;
+    left: 50%;
+    top: 3%;
+    transform: translateX(-50%);
+    border-radius: 40%;
+    height: $handHeightMin;
+    width: 5px;
+    background-color: black;
+    transform-origin: center bottom;
+    animation: moveHand 3600s infinite steps(60);
+  }
+
+  &__hand-sek {
+    position: absolute;
+    left: 50%;
+    top: 25%;
+    transform: translateX(-50%);
+    border-radius: 40%;
+    height: $handHeightSek;
+    width: 5px;
+    background-color: rgb(14, 57, 177);
+    transform-origin: center bottom;
+    animation: moveHand 60s infinite steps(60);
+  }
```

Comment:
You can add to hands one general class like - "watch_ hand" and put all of the repeated code there

Reply:
changed